src/algebra/ring/basic.lean

```diff
@@ -202,10 +202,14 @@ end add_monoid_hom
 
 /-- Bundled semiring homomorphisms; use this for bundled ring homomorphisms too. -/
 structure ring_hom (α : Type*) (β : Type*) [semiring α] [semiring β]
-  extends monoid_hom α β, add_monoid_hom α β
+  extends monoid_hom α β, add_monoid_hom α β, monoid_with_zero_hom α β
```

Comment:
Why not:
```suggestion
  extends monoid_with_zero_hom α β, add_monoid_hom α β
```
and declare a new function `ring_hom.to_monoid_hom`?

Reply:
Because if I do this this changes the field order from `⟨f, f_one, f_mul, f_zero, f_add⟩` to `⟨f, f_zero, f_one, f_mul, f_add⟩`, which is going to break a lot of code for no particularly good reason. The original order makes more sense to me anyway. I'll add a comment to that effect.